vowpalwabbit/cats.cc

```diff
@@ -144,8 +155,19 @@ void reduction_output::print_update_cb_cont(vw& all, const example& ec)
 LEARNER::base_learner* setup(options_i& options, vw& all)
 {
   option_group_definition new_options("Continuous actions tree with smoothing");
-  int num_actions = 0;
-  new_options.add(make_option("cats", num_actions).keep().necessary().help("number of tree labels <k> for cats"));
+  uint32_t num_actions = 0;
+  float bandwidth = 0;
+  float min_value = 0;
+  float max_value = 0;
+  new_options.add(make_option("cats", num_actions).keep().necessary().help("number of tree labels <k> for cats"))
+      .add(make_option("min_value", min_value).keep().help("Minimum continuous value"))
+      .add(make_option("max_value", max_value).keep().help("Maximum continuous value"))
+      .add(make_option("bandwidth", bandwidth)
+               .keep()
+               .help("Bandwidth (radius) of randomization around discrete actions in terms of continuous range. By "
+                     "default will be set to half of the continuous action unit-range resulting in smoothing that "
+                     "stays inside the action space unit-range:\nunit_range = (max_value - min_value) / "
+                     "num_actions\ndefault bandwidth = unit_range / 2.0"));
```

Comment:
Same comment as above here. Better to use the same terminology, "tree labels" or "actions"